google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobInfo.java

```diff
@@ -375,4 +376,23 @@ public static JobInfo of(JobId jobId, JobConfiguration configuration) {
   static JobInfo fromPb(Job jobPb) {
     return new BuilderImpl(jobPb).build();
   }
+
+  private static String getFieldAsString(Object field) {
+    return field == null ? "null" : field.toString();
+  }
+
+  protected Attributes getOtelAttributes() {
+    return Attributes.builder()
```

Comment:
of these, jobtype is probably the most valuable (if present) since it allows users to filter workloads.